_data/toc.yml

```diff
@@ -3,66 +3,121 @@
-          url: "docs/features/debugger"
-        - title: Enable/Disable Flags 
+        - title: Enabling/Disabling flag
           url: "docs/features/enable-disable-flags"
-        - title: Archive/Delete Flags 
+        - title: Default rule
+          url: "docs/features/default-rule"
+        - title: Targeting rules
+          url: "docs/features/targetingrules"
+        - title: Copy rules and configuration
+          url: "docs/features/copy-rule"
+        - title: Defining configuration for variation
+          url: "docs/features/variation-configuration"
+        - title: Analyze impact using Insights graph
+          url: "docs/features/insights-graph"
+        - title: Archive/Delete flags 
           url: "docs/features/archive-delete-flags"
+        - title: Flag settings
+          url: "docs/features/flag-settings"
         - title: Activity 
           url: "docs/features/audit-log"
-    - title: "SDKs"
-      url: "docs/sdks"
+    
+      - title: "Attributes"
+        url: "docs/attributes"
+        children:
+          - title: Defining attributes based on types
+            url: "docs/features/attributes"
+          - title: Operators based on attribute type
+            url: "docs/features/attributes-operators"
+          - title: Defining attributes in rules
+            url: "docs/features/attributes-in-rules"
+            
+    - title: "Projects and Environment"
+      url: "docs/projects"
       children:
-        - title: Client vs Server-side
-          url: "docs/sdks/client-vs-server-side-sdks"
-        - title: SDK Keys
+        - title: Creating projects and environments
+          url: "docs/features/projects-and-environments"
+        - title: API keys in project
           url: "docs/sdks/sdk-keys"
-        - title: Java SDK
-          url: "docs/sdks/java-sdk"
-        - title: JavaScript Library
-          url: "docs/sdks/javascript-library"
-        - title: Node.js SDK
-          url: "docs/sdks/nodejs-sdk"
-        - title: React SDK
-          url: "docs/sdks/react-sdk"
-        - title: Go SDK
-          url: "docs/sdks/go-sdk"
-        - title: .NET SDK
-          url: "docs/sdks/dotnet-sdk"
-        - title: Metrics - Impressions
-          url: "docs/sdks/metrics-impressions"
-        - title: Metrics - Insights
-          url: "docs/sdks/metrics-variationcounts"
-    - title: "Managing Users"
-      url: "docs/users"
-      children:
-        - title: Adding & Managing Users
+        - title: Teams attached to project
           url: "docs/users/team"
-        - title: Access Control
+        - title: Invite members
+          url: "docs/users/invite-member"
+        - title: Understanding member role
           url: "docs/users/access-control"
       
     - title: "Integrations"
       url: "docs/integrations"
       children:
         - title: Slack
           url: "docs/integrations/slack"
+        - title: Jira
+          url: "docs/integrations/jira"
+        - title: Sumologic
```

Comment:
"Sumo Logic" is how it's officially spelled

Reply:
Corrected